media/mathplayground/global.css

```diff
@@ -69,6 +69,20 @@ body,
     padding: 0.5em;
 }
 
+button {
+    background-color: transparent;
+    color:white;
+    border: none;
+}
```

Comment:
We shouldn't be defining button styles this generally - this will apply to all buttons on the page. And a lot of this we want to defer to Bootstrap anyways.

If you want to define styles for certain buttons your making I suggest using a class here.

Reply:
class made `.item-header`